Core.Collectors/Web/RequestResult.cs

```diff
@@ -6,19 +6,20 @@ namespace Microsoft.CloudMine.Core.Collectors.Web
 {
     public class RequestResult
     {
+        public HttpRequestMessage request;
```

Comment:
If this is accessed outside, this either needs to be property (with private set and public get) or there should be a method accessing it.